tests/drivers/counter/counter_basic_api/boards/frdm_k82f.conf

```diff
@@ -0,0 +1,2 @@
+CONFIG_COUNTER=y
+# CONFIG_COUNTER_MCUX_RTC=y
```

Comment:
`tests/drivers/counter/counter_basic_api/prj.conf` already sets `CONFIG_COUNTER=y`. You can remove this file

Reply:
ok